adot-testbed/app/build.gradle.kts

```diff
@@ -27,6 +27,16 @@ dependencies {
     testImplementation("com.github.rholder:guava-retrying:2.0.0")
     testImplementation("org.assertj:assertj-core:3.24.2")
     testImplementation("com.fasterxml.jackson.core:jackson-databind:2.13.0")
+
+    // Trace ID (W3C & XRay) tests with XRay Exporter
+    api(platform("io.opentelemetry:opentelemetry-bom:1.30.0"))
```

Comment:
if you use the alpha bom, you don't need to specify the version in any of the opentelemetry dependencies with gid `io.opentelemetry`

Reply:
changed to alpha bom